### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Comment by JustPies** [comment]
#docutune-ready

**Comment by lindalu-MSFT** [comment]
@JustPies this is still draft? I'm checking because I'm about to do a merge to live and wondered if you wanted to add this. Otherwise, it will be next Weds or Thurs.

**Comment by alexbuckgit** [comment]
@AlexJerabek @dstrome

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Comment by alexbuckgit** [comment]
Hi @lindalu-MSFT 

This is ready to go if you want to include in your merge. Thanks!

File: README.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-The live docs for this repo can be accessed at https://docs.microsoft.com/office/client-developer/office-client-development.
+The live docs for this repo can be accessed at [Office client development](/office/client-developer/office-client-development).
```
Comment (lindalu-MSFT) [inline README.md:5]:
```suggestion
Access the [Office client development](/office/client-developer/office-client-development) documentation on Microsoft Docs.
```

File: docs/access/desktop-database-reference/concepts.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ ms.localizationpriority: high
 
 This section provides important concepts for developing custom Access solutions.
 
-If you are viewing the content on docs.microsoft.com, you can use the table of contents located to the left of the topic to navigate through the documentation.
+If you are viewing the content on Microsoft Docs, you can use the table of contents located to the left of the topic to navigate through the documentation.
```
Comment (lindalu-MSFT) [inline docs/access/desktop-database-reference/concepts.md:18]:
```suggestion
If you're viewing the content on Microsoft Docs, use the table of contents on the left navigation pane to discover how-to content, sample code, SDK and API documentation, VBA references, training, and articles for developing solutions and customizing Office.
```

**Pull request merged**